common/persistence/cassandra/cassandraPersistenceTest.go

```diff
@@ -60,14 +60,18 @@ func NewTestCluster(keyspace, username, password, host string, port int, schemaD
 	if host == "" {
 		host = environment.GetCassandraAddress()
 	}
+	if protoVersion == 0 {
+		protoVersion = environment.GetCassandraProtoVersion()
```

Comment:
Would you like me to replace these  for the const `CassandraDefaultProtoVersionInteger` @longquanzheng ?

here, in `common/persistence/nosql/nosqlplugin/cassandra/tests/cassandra_tool_version_test.go` and `common/persistence/nosql/nosqlplugin/cassandra/tests/utils.go`, all in this diff.

Reply:
The tests have been using `environment.GetCassandraXYZ` so it's better to keep consistent. For others yes, if they were using the hardcoded value, then replace with the `CassandraDefaultProtoVersionInteger`
Thank you!